core/src/main/scala/org/locationtech/rasterframes/ref/GDALRasterSource.scala

```diff
@@ -78,7 +78,7 @@ object GDALRasterSource extends LazyLogging {
     val _ = new GDALWarp()
     true
   } catch {
-    case _: UnsatisfiedLinkError =>
+    case _:UnsatisfiedLinkError | _:NoClassDefFoundError =>
```

Comment:
@metasim  I suspect this may be a terrible idea.

Reply:
Hmmmmmm. Interesting......

Reply:
I don't understand why Travis isn't passing yet. CircleCI doesn't surprise me because GDAL is installed. I've not confirmed that it is in the Travis  configuration.